public/app/features/explore/Wrapper.test.tsx

```diff
@@ -154,8 +155,10 @@ describe('Wrapper', () => {
     (datasources.elastic.query as Mock).mockReturnValueOnce(makeLogsQueryResponse());
 
     // Make sure we render the logs panel
-    const logsPanels = await screen.findAllByText(/^Logs$/i);
-    expect(logsPanels.length).toBe(2);
+    await waitFor(() => {
+      const logsPanels = screen.getAllByText(/^Logs$/i);
+      expect(logsPanels.length).toBe(2);
+    });
```

Comment:
`findAllByText` will still return even though it finds only one panel, this makes it a bit more robust so it waits until both panels are rendered.